src/components/Books.js

```diff
@@ -0,0 +1,11 @@
+import './books.css';
+
+import React from 'react';
+
+const Books = () => (
+  <div className="books-container">
+    <h2>Books</h2>
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] Good job so far. However, there are some components that you need to create, and you need to also render the list of books on the browser as seen in the requirement. Below are what you need to do to be able to meet the requirement. I know you can do this and I appreciate your effort so far 👍.

- [ ] Please create a `BookList` component that does the following;
* Renders a list of books using the `Book` component
* Renders a form by using the `CreateNewBook` component

- [ ] Please create a `Book` component that does the following;
* Takes the book data as a prop and renders the `title` and the `author`
  * Has a button for removing books
    * the button wouldn't remove the books *yet*, this is just a placeholder for now

- [ ] Please create a `CreateNewBook` component that does the following;
* Renders a form with inputs for the `title` and the `author` of the book
* Has a button to submit
  * the button wouldn't add the book *yet*, this is just a placeholder for now

Note: You can name these components as you want but just keep the requirement for each component.

Reply:
done